docs/getting-started/integration.md

```diff
@@ -22,7 +22,21 @@ var myChart = new Chart(ctx, {...});
 
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
```

Comment:
is it weird that this refers to Chart.js 3? At the very least I think it should say "Chart.js 3+", but maybe we should change it to just "Chart.js"?